packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -21,8 +21,8 @@ const temporaryWordPressInternalTypes = [
 	'WPBlockTypeIcon',
 	'WPBlockTypeIconRender',
 	'WPBlockTypeIconDescriptor',
-	'WPComponent',
-	'WPElement',
+	'Component',
+	'Element',
```

Comment:
These are no longer custom types, right? So I believe they should be removed?

Reply:
This is basically related to https://github.com/WordPress/gutenberg/pull/54908#discussion_r1343754391 
Unless we make the effort to type everything properly, we can't remove these

Reply:
Makes sense. I guess we can add a comment about that.